### PR TITLE
Fix birthdate allowed formats

### PR DESCRIPTION
**Pull request by dahnbaez**

### CODE REVIEW

File: CmsWeb/healthcheck.txt [hunk 1]
```diff
@@ -1 +1 @@
-﻿down
\ No newline at end of file
+﻿downs
```
Comment (brondavies) [inline CmsWeb/healthcheck.txt]: This needs to be reverted

File: UtilityExtensions/Extensions/Dates.cs [hunk 2]
```diff
@@ -154,8 +154,10 @@ public static bool DateValid(string dt, out DateTime dt2)
         }
         public static bool BirthDateValid(string dob, out DateTime dt2)
         {
+            string[] formats = { "MM/dd/yyyy", "MM/d/yyyy", "M/dd/yyyy", "M/d/yyyy" };
```
Comment (brondavies) [inline UtilityExtensions/Extensions/Dates.cs]: this should also include `MM/d` and `MM/dd` for optional birth year.

File: CmsWeb/healthcheck.txt (hunk 1 above)
Comment (aternex) [reply CmsWeb/healthcheck.txt]: I will undo this, I just wanted to trigger a build quickly

File: UtilityExtensions/Extensions/Dates.cs (hunk 2 above)
Comment (brondavies) [reply UtilityExtensions/Extensions/Dates.cs]: You may have to handle those as a separate logic block lo differentiate how this function handles a date with and without a year.

Comment (dahnbaez) [reply UtilityExtensions/Extensions/Dates.cs]: mmm, but we need to make the birth year required, right?

File: UtilityExtensions/Extensions/Dates.cs [hunk 3]
```diff
@@ -154,8 +154,10 @@ public static bool DateValid(string dt, out DateTime dt2)
...
+
             dt2 = DateTime.MinValue;
-            if (DateTime.TryParseExact(dob, "m", CultureInfo.CurrentCulture, DateTimeStyles.None, out dt2))
+            if (!DateTime.TryParseExact(dob.ToString(), formats, CultureInfo.CurrentCulture, DateTimeStyles.None, out dt2))
```
Comment (brondavies) [inline UtilityExtensions/Extensions/Dates.cs]: the `!` is probably not what you want here and you also don't need the `ToString()`

**Pull request merged**